app/waf/src/models/common.model.ts

```diff
@@ -64,7 +66,7 @@ export abstract class CommonEntity extends Entity {
       response: true,
       example: 'My description',
       openapi: {
-        minLength: 1,
+        minLength: 0,
```

Comment:
What is the purpose to input zero length description?

Reply:
`minLength: 1,` will leads error, but in fact, zero-length description is anyhow valid, should be permitted.